verifier/verifier.go

```diff
@@ -176,6 +175,11 @@ func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelop
 		if err != nil {
 			return err
 		}
+        pluginVersion := metadata.Version
```

Comment:
Plugin version is not a valid semver, thats why adding "v" as prefix to it inside the function

Reply:
Its not valid semver, that's why we are adding "V" as prefix in the version compare fucntion

Reply:
As per spec(https://semver.org) a valid semver shouldn't contain `v` prefix. I'm assuming we are adding `v` because golang's library needs semver to be prefixed with `v`, am I missing something?

https://pkg.go.dev/golang.org/x/mod@v0.7.0/semver

Reply:
No you are right, for Compare functionality we are adding v